bindings/python/iota_sdk/utils.py

```diff
@@ -31,11 +32,11 @@ def bech32_to_hex(bech32: str) -> HexStr:
         })
 
     @staticmethod
-    def hex_to_bech32(hex: HexStr, bech32_hrp: str) -> str:
+    def hex_to_bech32(hex_addr: HexStr, bech32_hrp: str) -> str:
```

Comment:
`address: HexStr` ?

Reply:
Btw, aren't Python parameters named or something? Wouldn't it make it breaking change to rename them ? Excuse my python ignorance if not the case

Reply:
You may be right lol